scripts/canbench_ci_run_benchmark.sh

```diff
@@ -73,13 +73,10 @@ if [ -f "$MAIN_BRANCH_RESULTS_FILE" ]; then
   ' "$CANBENCH_OUTPUT" > "${CANBENCH_OUTPUT}.tmp" && mv "${CANBENCH_OUTPUT}.tmp" "$CANBENCH_OUTPUT"
 
   # Add a top-level summary of detected performance changes
-  if grep -q "(improved " "${CANBENCH_OUTPUT}"; then
-    echo "**🟢 Performance improvements detected! 🎉**" >> "$COMMENT_MESSAGE_PATH"
```

Comment:
Previously this script only detected either improvements or regressions. Now it can show both.